pulsar-transformations/src/main/resources/schema.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
maybe name the file `config-schema.json`

Reply:
sure